create_offline_installer.py

```diff
@@ -16,30 +16,38 @@
 
 # Pass the required miniconda installer version from devops pipelines variables
 def miniconda_installer_version():
-    return os.environ.get('MINICONDA_INSTALLER_VERSION', 'py37_4.8.3')
+    return os.environ.get('MINICONDA_INSTALLER_VERSION', 'py37_4.9.2')
 
 def required_offline_conda_packages():
     # these are the packages that we recommend for using the API
     # https://downloads.ccdc.cam.ac.uk/documentation/API/installation_notes.html#using-conda
+
+    # Please ensure that these versions are consistent with those in
+    # https://github.com/ccdc-confidential/cpp-apps-main/blob/main/wrapping/ccdc/requirements.txt
     api_pkgs = [
-        'pillow=7.0.0',
-        'six',
-        'lxml',
-        'numpy',
-        'matplotlib',
+        'pillow<9.0',
+        'six==1.16.0',
+        'lxml==4.6.3',
+        'numpy==1.20.3', # also used in mercury scripts
         'pytest',
-        'pandas',
-        'py-xgboost',
-        'scikit-learn',
+        'pandas==1.2.4', # also used in mercury scripts
+        'xgboost==1.4.0', # equivalent to py-xgboost, but more used
+        'scikit-learn==0.24.2',
     ]
```

Comment:
How does it behave with these exact versions on different platforms? I had issues previously with providing the exact version when switching between Linux and Win.

Reply:
I checked that these versions are carried on all three platforms by conda-forge :)

Reply:
> How does it behave with these exact versions on different platforms? I had issues previously with providing the exact version when switching between Linux and Win.

I checked that these versions are carried on all three platforms by conda-forge :)
Problem is: the generated miniconda doesn't have a condarc specifying it should pull stuff from conda-forge